Data_Science/StudentPerformanceVisualisation/README.md

```
@@ -0,0 +1,245 @@
+
+- **Python 3.x** - Core programming language
+- **Matplotlib** - Fundamental plotting library for creating static visualizations
+- **Seaborn** - Statistical data visualization library built on Matplotlib
+- **Pandas** - Data manipulation and analysis library
+- **NumPy** - Numerical computing library for handling arrays and mathematical operations
+- **Jupyter Notebook** - Interactive computing environment for creating and sharing documents
+- **SQL** (via Pandas) - For data querying and analysis in some notebooks
+
+## 📁 Repository Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)